sentry_sdk/utils.py

```diff
@@ -1563,6 +1566,26 @@ def parse_version(version):
     return release_tuple
 
 
+def integration_patched(original_function, integration=None):
```

Comment:
I would called it `ensure_integration_enabled(integration=None, original_function=None)` to make it easier to understand what is happening. (And if `original_function` is None, just fail with an error)